stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/ChatDomainImpl.kt

```diff
@@ -922,18 +907,19 @@ internal class ChatDomainImpl internal constructor(
     override fun deleteReaction(cid: String, reaction: Reaction): Call<Message> =
         client.deleteReaction(cid = cid, messageId = reaction.messageId, reactionType = reaction.type)
 
-    override fun markRead(cid: String): Call<Boolean> = MarkRead(this).invoke(cid)
-
-    override fun markAllRead(): Call<Boolean> = MarkAllRead(this).invoke()
-
-    override fun hideChannel(cid: String, keepHistory: Boolean): Call<Unit> =
-        HideChannel(this).invoke(cid, keepHistory)
-
-    override fun showChannel(cid: String): Call<Unit> = ShowChannel(this).invoke(cid)
+    override fun markRead(cid: String): Call<Boolean> {
```

Comment:
Any reason we didn't remove these when we removed these from`.livedata.ChatDomain`?

Reply:
yes, we want to remove everything but this method is being used somewhere or was being used once I prepared this PR 😢 
I'll revisit all the methods after migrating UI components